includes/reader-activation/class-reader-activation.php

```diff
@@ -1235,6 +1237,10 @@ public static function render_auth_form() {
 		}
 		?>
 		<div class="newspack-ui newspack-reader-auth">
+			<?php if ( false === $in_modal ) { ?>
```

Comment:
Nonblocking nit: If `$in_modal` will only ever be a boolean, this can be `! $in_modal`

Reply:
Good call! This has been fixed in [f70a4b9](https://github.com/Automattic/newspack-plugin/pull/3301/commits/f70a4b9f1ae2180e753799558ba0e34810950907).